backend/src/main/java/corea/room/controller/RoomControllerSpecification.java

```diff
@@ -40,8 +41,20 @@ ResponseEntity<RoomResponse> room(@Parameter(description = "방 아이디", exam
                                       long id,
                                       AuthInfo authInfo);
 
+    @Operation(summary = "방 참여 멤버들의 정보를 반환합니다.",
+            description = "상세 페이지에 디스플레이 되는 방 상세 정보를 반환합니다. <br>" +
+                    "요청 시 `Authorization Header`에 `Bearer JWT token`을 포함시켜야 합니다. " +
+                    "이 토큰을 기반으로 `AuthInfo` 객체가 생성되며 사용자의 정보가 자동으로 주입됩니다. <br>" +
+                    "JWT 토큰에서 추출된 사용자 정보는 피드백 작성에 필요한 인증된 사용자 정보를 제공합니다. " +
+                    "<br><br>**참고:** 이 API를 사용하기 위해서는 유효한 JWT 토큰이 필요하며, " +
+                    "토큰이 없거나 유효하지 않은 경우 인증 오류가 발생합니다.")
+    @ApiErrorResponses(value = {ExceptionType.MEMBER_NOT_FOUND, ExceptionType.ROOM_NOT_FOUND})
+    ResponseEntity<RoomMemberResponses> members(@Parameter(description = "방 아이디", example = "1")
```

Comment:
명세서 내용이 조금 이상하게 섞인 것 같습니다. 

`members` API와 `reviewers` API 명세 확인 부탁 드립니다! 

+) 별개로 조~~~금만 더 구체적인 설명이 달렸으면 좋겠어요!
(그대로 쓰셔도 됩니다.)

```suggestion
    @Operation(summary = "방 참가자들의 정보를 반환합니다.",
            description = "해당 방에 참여하고 있는 사람들 중 자신을 제외한 참여자들의 PR 링크를 랜덤으로 6명까지 노출합니다. <br>" +
                    "요청 시 `Authorization Header`에 `Bearer JWT token`을 포함시켜야 합니다. " +
                    "이 토큰을 기반으로 `AuthInfo` 객체가 생성되며 사용자의 정보가 자동으로 주입됩니다. <br>" +
                    "JWT 토큰에서 추출된 사용자 정보는 피드백 작성에 필요한 인증된 사용자 정보를 제공합니다. " +
                    "<br><br>**참고:** 이 API를 사용하기 위해서는 유효한 JWT 토큰이 필요하며, " +
                    "토큰이 없거나 유효하지 않은 경우 인증 오류가 발생합니다.")
    @ApiErrorResponses(value = {ExceptionType.MEMBER_NOT_FOUND, ExceptionType.ROOM_NOT_FOUND})
    ResponseEntity<RoomMemberResponses> members(@Parameter(description = "방 아이디", example = "1")
```

Reply:
아이구 명세를 잘못 바꿨었군요!
올바르게 수정했고, 말씀주신대로 설명 추가해 보았습니다!